### PR TITLE
Add scenario45 - DMS Kafka instance

### PR DESCRIPTION
**Pull request by danielaebert**
Create, List, Update, Delete DMS Kafka instance.

### CODE REVIEW

**Review by vladimirhasko — CHANGES_REQUESTED** [review]

File: playbooks/scenario45_dms_inst.yaml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+      - name: Create SG
+        openstack.cloud.security_group:
+          name: "{{ test_security_group_name }}"
+          description: "dummy"
+        register: newsg
+
+      - name: Create DMS Kafka Instance
+        opentelekomcloud.cloud.dms_instance:
+          name: "{{ dms_inst_name }}"
+          storage_space: '600'
```
[the remark this answers is not in the document]
Comment (danielaebert) [reply playbooks/scenario45_dms_inst.yaml]: Quotes removed

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

File: playbooks/scenario45_dms_inst.yaml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        - block:
+
+            - name: Delete DMS Kafka Instance
+              opentelekomcloud.cloud.dms_instance:
+                name: "{{ dms_inst_name }}"
+                state: absent
+
+            - name: Pause for 30 seconds
+              pause:
+                 seconds: 30
```
[the remark this answers is not in the document]
Comment (danielaebert) [reply playbooks/scenario45_dms_inst.yaml:88]: Unfortunately not.
The instance is deleted properly. checking the status returns "no instance found".
But the SG is still busy. I do not see another workaround than adding a pause statement.
Anyway, I have opened a bug for that:
https://jira.tsi-dev.otc-service.com/browse/BM-4558

**Review by anton-sidelnikov — DISMISSED** [review]

**Review by anton-sidelnikov — APPROVED** [review]

File: playbooks/scenario45_dms_inst.yaml [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+        register: newvpc
+
+      - name: Create subnet for VPC
+        opentelekomcloud.cloud.subnet:
+          name: "{{ test_subnet_name }}"
+          vpc_id: "{{ newvpc.vpc.id }}"
+          cidr: "10.10.0.0/27"
+          gateway_ip: "10.10.0.1"
+          dns_list:
+            - "100.125.4.25"
```
Comment (vladimirhasko) [inline playbooks/scenario45_dms_inst.yaml:51]: please check whether this is mandatory parameter.
If not you can remove dns_list at all

**Review by vladimirhasko — CHANGES_REQUESTED** [review]

**Comment by danielaebert** [comment]
New issue for DMS wait/timeout problem created:
https://github.com/opentelekomcloud/ansible-collection-cloud/issues/336

**Review by vladimirhasko — APPROVED** [review]

**Pull request merged**